Cargo.toml

```
@@ -146,13 +146,13 @@ unsigned-varint = { version = "0.7", default-features = false }
 url = { version = "2.3", features = ["serde"] }
 which = "4.3"
 
-fil_actor_account_state = "4"
-fil_actor_init_state = "4"
-fil_actor_interface = "4"
-fil_actor_miner_state = "4"
-fil_actor_power_state = "4"
-fil_actor_system_state = "4"
-fil_actors_shared = "4"
+fil_actor_account_state = "5"
+fil_actor_init_state = "5"
+fil_actor_interface = "5"
```

Version 5.1 has the devnet CIDs.

```suggestion
fil_actor_interface = "5.1"
```